project-fortis-interfaces/src/components/Admin/Streams/StreamConstants.js

```diff
@@ -260,77 +262,70 @@ const defaultStreamMap = {
     pipelineKey: 'Bing',
     pipelineLabel: 'Bing',
     pipelineIcon: 'Bing Icon',
-    streamFactory: 'Bing',
+    streamFactory: ['BingPage'],
     enabled: true
   },
-  EventHub: {
-    pipelineKey: 'EventHub',
-    pipelineLabel: 'EventHub',
-    pipelineIcon: 'EventHub Icon',
-    streamFactory: 'EventHub',
+  CustomEvent : {
+    pipelineKey: 'CustomEvent',
+    pipelineLabel: 'Custom Event',
+    pipelineIcon: 'Custom Event Icon',
+    streamFactory: ['CustomEvents'],
     enabled: true
   },
-  FacebookComment: {
-    pipelineKey: 'FacebookComment',
-    pipelineLabel: 'FacebookComment',
-    pipelineIcon: 'Facebook Comment Icon',
-    streamFactory: 'FacebookComment',
-    enabled: true
-  },
-  FacebookPost: {
-    pipelineKey: 'FacebookPost',
-    pipelineLabel: 'FacebookPost',
-    pipelineIcon: 'Facebook Post Icon',
-    streamFactory: 'FacebookPost',
+  Facebook: {
+    pipelineKey: 'Facebook',
+    pipelineLabel: 'Facebook',
+    pipelineIcon: 'Facebook Icon',
```

Comment:
Shouldn't these be [font-awesome](https://fontawesome.com/v4.7.0/icons/) class-names? Like `fa fa-facebook-official` or `fa fa-twitter` (the latter we use in some of the seed data).

Reply:
I'll look into that